docs-kits/kits/Eco_Pass_KIT/changelog.md

```diff
@@ -2,6 +2,18 @@
 title: Changelog
 ---
 
+## [released]
+## [2.0.0] - 2024-01-12
```

Comment:
Version needs to be changed to 1.0.0 and date needs to be adjusted to the release date: 2024-03-08

Reply:
done